lib/tools/adb-emu-commands.js

```diff
@@ -18,29 +18,37 @@ emuMethods.isEmulatorConnected = async function (udid) {
   return false;
 };
 
+emuMethods.checkEmulatorConnected = async function (udid) {
```

Comment:
@jlipps I was wondering that, should we always ask for an udid when using emulators ? I think so, right?